components/ai_chat/resources/page/components/page_context_toggle/style.module.scss

```diff
@@ -48,6 +45,20 @@
   padding: var(--leo-spacing-m);
 }
 
+.tooltipButton {
+  --leo-button-padding: 0;
+  width: 24px;
```

Comment:
Out of interest can you not use a `fab` button?

`<Button fab kind="plain-faint" size="tiny">`

You'll probably still need to set the icon color.

Reply:
does fab eliminate the need setting the button padding, is that all?

Reply:
It also make the button have a 1:1 aspect ratio